src/components/Inputs/ExchangeAmountInput/ExchangeAmountInput.tsx

```diff
@@ -77,37 +78,43 @@ export const AmountInput: React.FC<IProps> = ({
   }
 
   return (
-    <Input
-      inputRef={inputRef}
-      error={!!error}
-      className={classNames(classes.amountInput, className)}
-      classes={{ input: classes.input }}
-      style={style}
-      type={'text'}
-      value={value}
-      disableUnderline={true}
-      placeholder={placeholder}
-      onChange={allowOnlyDigitsAndTrimUnnecessaryZeros}
-      endAdornment={
-        <OutlinedButton
-          name='Max'
-          color='primary'
-          onClick={onMaxClick}
-          className={classes.maxButton}
-          labelClassName={classes.label}
-          disabled={disabled}
-        />
-      }
-      startAdornment={
-        <Select
-          centered={true}
-          tokens={tokens}
-          onSelect={onSelect}
-          current={current}
-          className={classes.select}
-        />
-      }
-    />
+    <>
+      <Input
+        inputRef={inputRef}
+        error={!!error}
+        className={classNames(classes.amountInput, className)}
+        classes={{ input: classes.input }}
+        style={style}
+        type={'text'}
+        value={value}
+        disableUnderline={true}
+        placeholder={placeholder}
+        onChange={allowOnlyDigitsAndTrimUnnecessaryZeros}
+        startAdornment={
+          <Select
+            centered={true}
+            tokens={tokens}
+            onSelect={onSelect}
+            current={current}
+            className={classes.select}
+          />
+        }
+      />
+      <Box className={classes.container}>
```

Comment:
about data that should be shown at bottom of the input: look at the input used in new position view in don it in similar way